src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -742,6 +751,9 @@ void DlgPrefSound::settingChanged() {
     m_settingsModified = true;
 
 #ifdef __RUBBERBAND__
+}
+
+void DlgPrefSound::updateKeylockDualThreadingCheckbox() {
```

Comment:
This `#ifdef` looks suspicious, does it work as intended? I'm getting some errors building for iOS (without Rubberband)...

Reply:
Seems to work, but the brace placement makes this confusing. I've fixed this and some related issues in

- https://github.com/mixxxdj/mixxx/pull/13577